src/modules/commander/commander.cpp

```diff
@@ -2881,7 +2881,7 @@ int commander_thread_main(int argc, char *argv[])
 						       _mission_result.stay_in_failsafe,
 						       &status_flags,
 						       land_detector.landed,
-						       (rc_loss_enabled > 0),
+						       rc_loss_mode,
```

Comment:
Note that the same param is used again in the navigator which is really quite ugly:
https://github.com/PX4/Firmware/blob/master/src/modules/commander/commander.cpp#L1276
https://github.com/PX4/Firmware/blob/master/src/modules/navigator/navigator_main.cpp#L162